src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Serialization/RazorConfigurationJsonConverter.cs

```diff
@@ -60,4 +91,50 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
             writer.WriteEndObject();
         }
     }
+
+    internal class RazorLanguageVersionObjectJsonConverter : JsonConverter
```

Comment:
Didn't separate into a different file as I want to keep this logic very isolated. This shouldn't be used as a normal converter or anything, and is simply meant to handle reading the MVC 1.1 language version object.

```
{
  "ConfigurationName": "MVC-1.1",
  "Extensions": [
    {
      "ExtensionName": "MVC-1.1"
    }
  ],
  "LanguageVersion": {
    "Major": 1,
    "Minor": 1
  }
}
```